code/modules/mining/mine_items.dm

```diff
@@ -243,7 +244,9 @@
 	threshhold.nitrogen = 82
 	threshhold.carbon_dioxide = 0
 	threshhold.toxins = 0
-	L.contents += threshhold
+	var/area/ZZ = get_area(threshhold)
+	if(!(ZZ.type in blacklist))

```

Comment:
This won't work with subtypes of `/area/shuttle`. Use `if(!is_type_in_list(ZZ, blacklist))`